vignettes/articles/06-exploring-data.Rmd

```diff
@@ -101,18 +103,21 @@ sql.motus %>%
   select(ts, motusTagID, runLen, speciesEN, tagDepLat, tagDepLon, 
          recvDeployLat, recvDeployLon) %>% 
   collect() %>%
+  mutate(ts = as_datetime(ts)) %>% 
   summary()
 
 # same summary for the filtered sql data
 df.alltags.sub %>% 
   select(ts, motusTagID, runLen, speciesEN, tagDepLat, tagDepLon, 
-         recvLat, recvLon) %>% 
+         recvDeployLat, recvDeployLon) %>% 
```

Comment:
Above (~ L58) the `df.alltags.sub` is created with the new fields `recvLat` and `recvLon` and `recvAlt`, which are `gps` values if we have them and `recvDeploy` values if we don't. I definitely find this confusing. Is it worth keeping that in, here? Or should I move the lines where `recvLat` etc. are created to the GPS article, and just use `recvDeployLat` etc. here and below?

Right now this errors, however, because `recvDeployLat` doesn't exist.

Reply:
My personal preference is to just use `recvDeploy` coordinates, as we have done throughout the complete walkthrough chapters. Adding in `gpsLat` and `gpsLon` fields in earlier seems like it could cause unnecessary confusion and would fit better in the GPS article.

Reply:
I agree 100%, thanks!